src/Files.App/Views/Settings/GeneralPage.xaml

```diff
@@ -120,66 +120,73 @@
 				</ComboBox>
 				<local:SettingsBlockControl.ExpandableContent>
 					<StackPanel>
-						<local:SettingsBlockControl HorizontalAlignment="Stretch" Visibility="{x:Bind ViewModel.OpenASpecificPageOnStartup, Mode=OneWay}">
-							<Grid HorizontalAlignment="Stretch">
-								<Grid.ColumnDefinitions>
-									<ColumnDefinition Width="*" />
-									<ColumnDefinition Width="Auto" />
-								</Grid.ColumnDefinitions>
-								<Grid.RowDefinitions>
-									<RowDefinition />
-									<RowDefinition />
-								</Grid.RowDefinitions>
-								<TextBlock
-									HorizontalAlignment="Left"
-									VerticalAlignment="Center"
-									FontSize="16"
-									Text="{helpers:ResourceString Name=SettingsOnStartupPages/Text}" />
-								<CommandBar
-									Grid.Column="1"
-									Background="Transparent"
-									DefaultLabelPosition="Right"
-									IsOpen="False">
-									<AppBarButton Icon="Add" Label="{helpers:ResourceString Name=Add}">
-										<AppBarButton.Flyout>
-											<MenuFlyout helpers:MenuFlyoutHelper.ItemsSource="{x:Bind ViewModel.AddFlyoutItemsSource, Mode=OneWay}" Placement="Bottom" />
-										</AppBarButton.Flyout>
-									</AppBarButton>
-									<AppBarButton
-										Command="{x:Bind ViewModel.ChangePageCommand}"
-										Icon="Edit"
-										IsEnabled="{x:Bind ViewModel.IsPageListEditEnabled, Mode=OneWay}"
-										Label="{helpers:ResourceString Name=Edit}" />
-									<AppBarButton
-										Command="{x:Bind ViewModel.RemovePageCommand}"
-										Icon="Delete"
-										IsEnabled="{x:Bind ViewModel.IsPageListEditEnabled, Mode=OneWay}"
-										Label="{helpers:ResourceString Name=Remove}" />
-								</CommandBar>
+						<Grid
+							Padding="8"
+							HorizontalAlignment="Stretch"
+							RowSpacing="4">
+							<Grid.RowDefinitions>
+								<RowDefinition Height="Auto" />
+								<RowDefinition Height="*" />
+							</Grid.RowDefinitions>
+							<Button Margin="12,0,0,0">
+								<Button.Flyout>
+									<MenuFlyout Placement="RightEdgeAlignedBottom">
+										<MenuFlyoutItem
+											Command="{x:Bind ViewModel.AddPageCommand}"
+											CommandParameter="Home"
+											Text="{helpers:ResourceString Name=Home}" />
+										<MenuFlyoutItem Command="{x:Bind ViewModel.AddPageCommand}" Text="{helpers:ResourceString Name=Browse}" />
+									</MenuFlyout>
+								</Button.Flyout>
+								<StackPanel Orientation="Horizontal" Spacing="8">
+									<FontIcon
+										Margin="0,1,0,0"
```

Comment:
Is this margin needed? It's a bit of an odd number..

Reply:
The + icon looks off centre without it

Reply:
Does a vertical alignment help?

Reply:
Unfortunately not, the extra space seems to be part of the glyph itself

Reply:
The issue with using `1` is that it'll only work in some scales.

Reply:
Yeah, it's tempting to "manually" tweak margins like this, but generally: just don't. Especially not with odd numbers like this.